modules/claims_api/app/sidekiq/claims_api/service_base.rb

```diff
@@ -61,11 +61,15 @@ def save_auto_claim!(auto_claim, status)
 
     def set_evss_response(auto_claim, error)
       auto_claim.status = ClaimsApi::AutoEstablishedClaim::ERRORED
```

Comment:
@stiehlrod  I believe this line won't be needed anymore since this will get set before coming to this part.

Reply:
Oh actually this is odd,  I see it here in the PR but not when I pull down the branch, so you can probably ignore that if you did remove that